homeassistant/components/control4/light.py

```diff
@@ -124,9 +124,9 @@ async def async_update_data_dimmer():
                 item_coordinator,
                 item_name,
                 item_id,
-                item_device_name,
-                item_manufacturer,
-                item_model,
+                str(item_device_name),
+                str(item_manufacturer),
+                str(item_model),
```

Comment:
This ends up being used in the device info, which actually can accept `None` values. The correction done here is indeed satisfying MyPy, by it is not resolving the problem correctly.

The problem is that the typing down the line is incorrect, which should be corrected instead of casting to a string.

Reply:
Agreed, I just have a non-successful experience changing "down the line", I will mark this as draft, and make a PR to change the originating class.

Reply:
Solved, thanks for catching this !!